beam/README.md

```diff
@@ -8,12 +8,9 @@ Current interpreter implementation supports the static repl. It compiles the cod
 You have to first build the Beam interpreter by enable the **beam** profile as follows:
 
 ```
-mvn clean package -Pbeam -DskipTests
+mvn clean package -Pbeam -DskipTests -Pscala-2.10
```

Comment:
Thanks @1ambda . I verified the libraries in Beam, that it only supports scala_2.10 so far.